protocol/errors.json

```diff
@@ -182,6 +182,39 @@
   "91004": "unable to automatically re-enter presence channel",
   "91005": "presence state is out of sync",
   "91100": "member implicitly left presence channel (connection closed)",
+  "92000": "Invalid MapOp: key required",
+  "92001": "Channel state max objects limit exceeded",
+  "92002": "Channel message must contain state operation message",
+  "92003": "Invalid state message object ID length",
```

Comment:
Do you think we need to be this granular or is it worth having a single `Invalid state operation` error and we add additional context in the error returned to the client?